### PR TITLE
Save text area sizes for sessions

### LINKED ISSUE #80: Save form resize settings
When using the form, I often resize fields like "Summary" or "Comments"

I don't like having to resize every time I visit the form. If resized, it should just stay resized on the next visit

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #80

### CODE REVIEW

File: extension/ts/extensions/resize.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	id: string;
+}
+
+interface Size {
+	width: string;
+	height: string;
+}
+
+interface SizeData {
+	[id: string]: {width: string; height: string};
```
Comment (adirar111) [inline extension/ts/extensions/resize.ts:28]: this value's type and `Size` look the same, is there a reason we're not doing `[id: string]: SIze`?

File: extension/ts/extensions/resize.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+const observer = new ResizeObserver((entries) =>
```
Comment (adirar111) [inline extension/ts/extensions/resize.ts:31]: this is a thing? the more you know💫

File: extension/ts/extensions/resize.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+const ifTextArea =
+	(callback: (element: HTMLTextAreaElement) => void) =>
+	(element: Element): void =>
+		element.tagName.toUpperCase() === "TEXTAREA" && callback(element as HTMLTextAreaElement);
+
+const beenAWhile = () => getLastFormRender() + ONE_DAY_MS < Date.now();
+const clearSizeData = () => localStorage.setItem(SIZE_LOCAL_STORAGE_KEY, "{}");
+
+onFormRender((form: HTMLElement, forEachElement: ForEachFormElement) => {
+	beenAWhile() && clearSizeData();
```
Comment (adirar111) [inline extension/ts/extensions/resize.ts:93]: AHAHAHA `beenAWhile` is a 10/10 function name

**Review by adirar111 — APPROVED** [review]
yay to no more constant manual resizing

File: extension/ts/extensions/resize.ts (hunk 1 above)
Comment (braxtonhall) [reply extension/ts/extensions/resize.ts:28]: good catch my b

File: extension/ts/extensions/resize.ts (hunk 3 above)
Comment (braxtonhall) [reply extension/ts/extensions/resize.ts:93]: lmao functions a descended from labels are descended from comments 💆‍♀️

**Pull request merged**